coursebuilder/modules/explorer/_static/js/course.js

```diff
@@ -15,7 +15,9 @@ $(document).ready(function documentReady() {
   });
 
   $('.course-carousel').on('changed.owl.carousel', function(event) {
-    window.scrollTo(20, 20);
+    var el = $(".course-carousel");
+    var position = el.position();
```

Comment:
@Sharonsyra are there cases where el.length > 1 ?

Reply:
Yes. But in this case no since the div with the class name appears only once.

Reply:
@Sharonsyra let us design for edge cases e.g https://courses.academy.africa/test/unit?unit=1&lesson=20

Reply:
@chegejames I don't have access to it.

Reply:
@Sharonsyra https://cfafrica-mooc-dev.appspot.com/test/unit?unit=4&lesson=11

Reply:
Hey @chegejames still working on this. There is a bug. window.scrollTo() is always scrolling to the top even when the coordinates specify a different position in the page. Console.log() returns the correct position depending on the carousel clicked on though.

Reply:
@chegejames this works perfectly now. You can test it out here. https://cfafrica-mooc-dev.appspot.com/test/unit?unit=4&lesson=13

Reply:
@Sharonsyra do all browsers support / implement 'this'?